.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+# Description
+
+<!-- Please use this template to file a pull request for opta and provide as much info as possible. 
+We'll be making script based off this structure for releases and will be annoyed if it is not followed.
```

Comment:
Don't say this - it doesn't sound welcoming to external contributors. I think you can just remove this whole comment.